app.js

```diff
@@ -1,8 +1,89 @@
-// import functions and grab DOM elements
+const button1 = document.getElementById('shell-1-button');
+const button2 = document.getElementById('shell-2-button');
+const button3 = document.getElementById('shell-3-button');
 
-// let state
+const shell1 = document.getElementById('shell-1');
+const shell2 = document.getElementById('shell-2');
+const shell3 = document.getElementById('shell-3');
 
-// set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+const winSpan = document.getElementById('wins');
+const lossSpan = document.getElementById('losses');
+const totalSpan = document.getElementById('total');
+
+let wins = 0;
+let losses = 0;
+
+button1.addEventListener('click', () => {
+    shell1.classList.remove('reveal');
```

Comment:
Since we're repeating this removal of the "reveal" class, this can be abstracted into its own function that we can call within each button